README.md

```diff
@@ -54,68 +51,63 @@ search.web('Tutta Bella Neapolitan Pizzeria',
 
 Output:
 ```javascript
-[ { id: 'e66c09f5-3317-4227-aee7-bc5dec3aec43',
-    title: 'Tutta Bella Neapolitan Pizzeria',
-    description: 'Neapolitan pizzeria located...',
-    displayUrl: 'tuttabellapizza.com',
-    url: 'http://tuttabellapizza.com/' },
-  { id: '94b5a08a-6148-4832-9077-98245f3e1b42',
-    title: 'Tutta Bella Neapolitan Pizzeria - Columbia City',
-    description: 'Tutta Bella Columbia City...',
-    displayUrl: 'www.tuttabella.com/tuttabellacc/index.php',
-    url: 'http://www.tuttabella.com/tuttabellacc/index.php' },
-  { id: '11bf839d-ce9e-4215-9b55-d663bdd17b98',
-    title: 'Tutta Bella Menu - Tutta Bella Neapolitan Pizzeria',
-    description: 'Tutta Bella Neapolitan Pizzeria - Columbia City...',
-    displayUrl: 'www.tuttabella.com/menu',
-    url: 'http://www.tuttabella.com/menu/' },
-  { id: 'da706086-5566-414b-b70f-2648e762cdf6',
-    title: 'Tutta Bella Neapolitan Pizzeria - Crossroads',
-    description: 'Ciao! Come visit us in our newest Tutta Bella location...',
-    displayUrl: 'tuttabella.com/tuttabellacr/index.php',
-    url: 'http://tuttabella.com/tuttabellacr/index.php' },
-  { id: '4d4d0961-bdd1-406f-aa80-39d067318c49',
-    title: 'Tutta Bella Neapolitan Pizzeria - Westlake',
-    description: 'Parking. Tutta Bella validates guest parking...',
-    displayUrl: 'www.tuttabella.com/TuttaBellaWL',
-    url: 'http://www.tuttabella.com/TuttaBellaWL/' } ]
+[ { title: 'Tutta Bella',
```

Comment:
Thanks! I wasn't parsing out the `id` originally, and forgot to update this part of the README.